packages/sdk/src/Base.ts

```diff
@@ -164,6 +164,11 @@ class Base {
     }
   }
 
+  sendTransaction (transactionRequest: any, chain: TChain) {
```

Comment:
minor suggestion: set type to ethers `TransactionRequest`

```suggestion
  sendTransaction (transactionRequest: TransactionRequest, chain: TChain) {
```

Reply:
or `transactionRequest: TransactionRequest | ContractTransaction`

Reply:
thanks :+1: 

Resolved by https://github.com/hop-protocol/hop/pull/412/commits/1b993a50ec48a897213310a8e03f7a0f08c6c058